lib/virt_autotest/utils.pm

```diff
@@ -85,19 +104,45 @@ sub restart_modular_libvirt_daemons {
     } else {
         # Restart the sockets first
         foreach my $daemon (@daemons) {
-            systemctl("restart $daemon\{,-ro,-admin\}.socket");
+            if ($daemon == 'lock') {
+                systemctl("restart virt${daemon}d\{,-admin\}.socket");
+            } else {
+                systemctl("restart virt${daemon}d\{,-ro,-admin\}.socket");
+            }
         }
 
         # Introduce idle time here (e.g., sleep 5) if necessary
         sleep 5;
```

Comment:
Can you help improve this to get rid of ```sleep 5``` ?  I think what needs to be checked is quite clear inside this function, ```sleep``` should not be used.

Reply:
likewise, try to use `script_retry()` instead of `sleep` here.

Reply:
Refer to the latest validation run - http://10.67.129.59/tests/91#step/update_package/150 use with `script_retry()` instead of `sleep` works better now.